gpii/node_modules/preferencesServer/src/oldPreferencesHandlers.js

```diff
@@ -13,28 +13,24 @@ You may obtain a copy of the License at
 https://github.com/GPII/universal/blob/master/LICENSE.txt
```

Comment:
Just an FYI: I think we can remove all this stuff -- I created a JIRA for doing so (https://issues.gpii.net/browse/GPII-1556).. In either case, I dont think it's necessary to be done as part of this pull